homework/Vasili_Zasinets/src/main/java/com/tutrit/java/quickstart/homework/set/UniqueWordVocabulary.java

```diff
@@ -0,0 +1,51 @@
+package com.tutrit.java.quickstart.homework.set;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.LinkedHashSet;
+import java.util.Objects;
+import java.util.Set;
+
+public class UniqueWordVocabulary {
+
+    Logger log = LoggerFactory.getLogger("Set ");
```

Comment:
I write eyarly what space in "Set " is the  crutch.

Reply:
fixed